web/src/javascript/game.js

```diff
@@ -10,12 +10,12 @@ import {
 const apiURL = "http://localhost:1337/api/1/game/";
 
 export function draw(gameState, canvas) {
-  initMap(canvas);
+  initMap(canvas, gameState.players.length);
```

Comment:
we could just pass `gameState.map` instead of `players` here, as per my previous comment. :-)